lib/git.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
```

Comment:
I would only put on `.gitattributes` that lib is generated code, so reviews are easier:

```gitattributes
lib/*.js linguist-generated=true
```

Reply:
Didn't know about this flavor, thanks!